app/helpers/aws_helper.rb

```diff
@@ -0,0 +1,47 @@
+module AwsHelper
+  def presigned_image_upload_url(image_type:, transaction_id:)
+    s3_presigned_url(
+      bucket_prefix: 'login-gov-idp-doc-capture',
+      keyname: "#{transaction_id}-#{image_type}",
+    )
+  end
+
+  def s3_presigned_url(bucket_prefix:, keyname:)
+    raise(ArgumentError, 'keyname is required') if keyname.nil?
+    return nil unless s3_resource
+
+    obj = s3_resource.bucket(bucket(prefix: bucket_prefix)).object(keyname)
+    URI.parse(obj.presigned_url(:put))
+  end
+
+  def s3_resource
+    Aws::S3::Resource.new(region: aws_region)
+  rescue Aws::Sigv4::Errors::MissingCredentialsError => aws_error
+    Rails.logger.info "Aws Missing CredentialsError!\n" + aws_error.message
+    nil
+  end
```

Comment:
So my thought was for local dev, create a test controller (like our `Test::SamlController` or something) and pass a URL for that instead. Do we want to do that in this PR or in a followup?